utils/zetachain.command.helpers.ts

```diff
@@ -238,34 +237,28 @@ export const addCommonZetachainCommandOptions = (command: Command) => {
         .default(DEFAULT_ACCOUNT_NAME)
         .conflicts(["private-key"])
     )
-    .addOption(
-      new Option("--network <network>", "Network to use")
-        .choices(["mainnet", "testnet"])
-        .default("testnet")
-    )
+    .addOption(new Option("--chain-id <chainId>", "Chain ID of the network"))
```

Comment:
This used to offer choices on the previous implementation (`mainnet` | `testnet`), shouldn't we still offer the same choices but in the form of chain ids? (`"7000"` | `7001`)

Reply:
Instead of choices for something like this, I'd maybe look into helper functions or aliases.

Reply:
We could have a refine rule that checks if the provided chain id complies with the available chain (i.e.: if the command is supposed to run on EVM, then it should only accept EVM compliant chain ids).